examples/service.yaml

```diff
@@ -87,6 +91,7 @@ topology_template:
         image: 9ea4856a-32b2-4553-b408-cfa4cb1bb40b
         flavor: d3046a41-245a-4042-862e-59568e81f8fa
         network: 753940e0-c2a7-4c9d-992e-4d5bd71f85aa
+        security_groups: default
```

Comment:
If we decide to make `security_groups` a list of strings, this will need to be adjusted accordingly.